### ISSUE TITLE
Support regex in enums

### ISSUE THREAD

**Issue by chadfurman** [mjmlio/mjml#2151]
**Is your feature request related to a problem? Please describe.**
Enums could easily support regex if you remove the regex escape

**Describe the solution you'd like**
I should be able to specify a custom validator like `enum(auto,[\d]+px)` for width properties etc

**Comment by iRyusa** [comment]
Might be a bit hard to parse them tho, why won't you use `unit` instead ?

**Comment by chadfurman** [comment]
Ah that's neat, didn't know you could use auto with unit, thank you :)
